.evergreen/run-checks.sh

```diff
@@ -1,9 +1,21 @@
 #!/bin/bash
-set -o errexit  # Exit the script with error if any of the commands fail
+set -o errexit # Exit the script with error if any of the commands fail
 
 export PROJECT_DIRECTORY="$(pwd)"
 NODE_ARTIFACTS_PATH="${PROJECT_DIRECTORY}/node-artifacts"
 export NVM_DIR="${NODE_ARTIFACTS_PATH}/nvm"
 [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
 
+set -o xtrace
```

Comment:
Was this meant to stay here or is it left over from development? I find it generally clutters up logs having this on (and it can leak sensitive data).

Reply:
Just testing, now that I have the useful "echo"s we dont need this